data/mods/My_Sweet_Cataclysm/sweet_comestibles.json

```diff
@@ -151,5 +151,28 @@
     "calories": 152,
     "description": "Dehydrated chocolate milk powder.  Mix with water to make drinkable milk.",
     "fun": -1
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "licorice",
```

Comment:
Please add a regular candy version of this in vanilla. The idea is that the candy that spawns in vanilla now turn into monster with that mod so when adding new candy it's better to have the regular candy spawn in vanilla and then make an override in MSC that change the description a tiny bit and add the "rot_spawn' entry to make them spawn the candy monster

Reply:
Don't know how to do this in vanilla and don't wanna break anything.